Update index.md (#4912)
segmentio/segment-docs: src/connections/storage/catalog/redshift/index.md

### Do you support Redshift Serverless?

Segment does not currently support Serverless Redshift. While you can set up the connection in the Segment app, Segment does not have the functionality to query Redshift's SYS tables.

> ### What is the recommended approach for changing the Host of an existing Redshift destination connection?

Reviewer: ```suggestion
### How can I change the host for an existing Redshift destination?
```

> The recommended approach to do this would be first to disable the sync for the warehouse, check that the syncs that are in progress have stopped/finished, and then update the host. Once updated, you can then re-enable the sync.

Reviewer: ```suggestion
Segment recommends that you first disable the sync for the warehouse, check that the syncs that are in progress have stopped/finished, and then update the host. Once you've updated the host, you can then re-enable the sync.
```
